block/src/Pos/Block/Logic/Header.hs

```diff
@@ -223,7 +223,8 @@ classifyHeaders inRecovery headers = do
         CHsUseless "Couldn't find lca -- maybe db state updated in the process"
     processClassify tipHeader = runMaybeT $ do
         lift $ logDebug $
-            sformat ("Classifying headers: "%listJson) $ map (view headerHashG) headers
+            sformat ("Classifying headers (newest first): "%buildListBounds) $
+                getNewestFirst $ map (view headerHashG) headers
```

Comment:
Do we still using `formatting` and not `fmt`?

Reply:
*Shrug* The code I was changing clearly uses `formatting`.